pkg/agent/route/route_linux.go

```diff
@@ -94,7 +95,9 @@ type Client struct {
 	nodeConfig    *config.NodeConfig
 	networkConfig *config.NetworkConfig
 	noSNAT        bool
-	ipt           *iptables.Client
+	ipt           iptables.Interface
```

Comment:
Personally I'm fine with either but the variable was named `ipt` since the begining and the PR is for unit testing, not naming style adjustment, so I would prefer keeping it as was to avoid unrelated change.